app-backend/rest-api/package.json

```diff
@@ -12,13 +12,9 @@
     "deploy-prod": "serverless --stage=production deploy"
   },
   "dependencies": {
-    "apollo-server-lambda": "1.3.2",
-    "apollo-tracing": "^0.0.9",
+    "apollo-server-lambda": "^2.2.1",
     "babel-polyfill": "^6.26.0",
-    "graphql": "^0.10.5",
-    "graphql-playground-middleware": "^1.1.0",
```

Comment:
don't we need playground packages now?

Reply:
nope both tracing and playground packages are included in `apollo-server-lambda` itself